cdisc-org/conformance-rules-editor
# ISS 60: Added export button

To test:
Run rule editor
Select a rule
Add test data and define xml
Click the new `Export Artifacts` button (down arrow in top left corner).

The browser should create and download a new zip file with the following contents:
- Datasets.json
- Datasets.xlsx
- Define.xml
- Request.json
- Results.json
- Rule.yml
- Rule_spaces.json
- Rule_underscores.json
- Schema_Validation.json
